Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -82,6 +88,20 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager, ShiftSub
         previousRightTicks = rightDrive.getPosition();
         leftInchesTraveled = 0;
         rightInchesTraveled = 0;
+        
+        drivingStateLatch.addObserver((Observable o, Object arg) -> {
+            if(arg instanceof EdgeType) {
+                EdgeType edge = (EdgeType)arg;
+                String type = "";
+                if(edge == EdgeType.RisingEdge) {
+                    type = "starting";
+                }
+                else if(edge == EdgeType.FallingEdge) {
+                    type = "stopping";
+                }
+                log.info(String.format("Drive %s left: %.2f, right: %.2f", type, this.leftDrive.get(), this.rightDrive.get()));
```

Comment:
I agree periodic logging would be nice too, I think I'd want this in addition to that but if you think it's net distracting to have the extra information there I'm happy to remove it since you'll be the one looking at it next week ;)